features/reset/reset_and_delete/feature.go

```diff
@@ -0,0 +1,174 @@
+package reset_and_delete
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"go.temporal.io/api/common/v1"
+	"go.temporal.io/api/enums/v1"
+	historyProto "go.temporal.io/api/history/v1"
+	"go.temporal.io/api/serviceerror"
+	"go.temporal.io/api/workflowservice/v1"
+	"go.temporal.io/features/harness/go/harness"
+	"go.temporal.io/sdk/client"
+	"go.temporal.io/sdk/workflow"
+)
+
+const (
+	SignalName = "drivesignal"
+)
+
+var Feature = harness.Feature{
+	Workflows:   Workflow,
+	Execute:     Execute,
+	CheckResult: CheckResult,
+	// We do this kinda by hand in check result -- it's easier since we want to pass the new run
+	// to check result, but end up looking at the history of the original run
+	CheckHistory: func(ctx context.Context, runner *harness.Runner, run client.WorkflowRun) error {
+		return nil
+	},
+}
+
+var originalRunID string
+
+func Execute(ctx context.Context, r *harness.Runner) (client.WorkflowRun, error) {
+	// Start the workflow
+	run, err := r.ExecuteDefault(ctx)
+	originalRunID = run.GetRunID()
+
+	if err != nil {
+		return nil, err
+	}
+	// Drive it for a bit
+	for i := 0; i < 5; i++ {
+		err := r.Client.SignalWorkflow(ctx, run.GetID(), run.GetRunID(), SignalName, "")
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	time.Sleep(1 * time.Second)
+	// Find event ID of second WFT complete
```

Comment:
just want to confirm that there's nothing special about the second WFT complete right?

Reply:
No, nothing special. Just had to pick one.